CMakeLists.txt

```diff
@@ -206,7 +206,9 @@ if (NOT CLAZY_BUILD_WITH_CLANG)
   install(FILES README.md COPYING-LGPL2.txt checks.json DESTINATION ${DOC_INSTALL_DIR})
 
   # Build docs
-  set(MAN_INSTALL_DIR "man/man1")
+  if(NOT MAN_INSTALL_DIR)
```

Comment:
I believe what we really want is

set(DATAROOTDIR "${CMAKE_INSTALL_DATAROOTDIR}" CACHE PATH "Define install directory for read-only architecture-independent data")
set(MAN_INSTALL_DIR "${DATAROOTDIR}/man/man1")

Reply:
I just found `SHARE_INSTALL_DIR` is already defined: https://github.com/KDE/clazy/pull/20/files#diff-af3b638bc2a3e6c650974192a53c7291L182
And PR is updated to make use of it.